core/src/smartcontracts/wasm.rs

```diff
@@ -526,18 +628,133 @@ mod tests {
             isi1_end = isi_hex.len() / 3,
             isi2_end = 2 * isi_hex.len() / 3,
         );
-        let mut runtime = Runtime::from_configuration(Configuration {
-            fuel_limit: 100_000,
-            max_instruction_count: 1,
-        })?;
-        let res = runtime.execute(&wsv, account_id, wat);
+
+        let mut runtime = Runtime::new()?;
+        let res = runtime.validate(
+            &wsv,
+            &account_id,
+            wat,
+            1,
+            Arc::new(AllowAll.into()),
```

Comment:
I like that `AllowAll` is now explicit. It would be nice to have 
```
fn allow_all() -> Arc<AllowAll> {
	return Arc::new(
	// explicit conversions here
	)
}

Reply:
yeah, it would remove some bloat